### ISSUE TITLE
BUG: `k-build` silently doesn't work if `k.config.mjs` is incorrect

### ISSUE THREAD

**Issue by Riernar** [Kurohyou-Studios/k-scaffold#95]
If the `k.config.mjs` file is incorrect (e.g contains a `require()`, which seems not to work in `.mjs` files), then `k-build` does nothing without raising any error.

Version `2.2.2` also stopped working with `k.config.js` (note the extension), where a warning is printed:
```
(node:8141 Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
```
and then `k-build` also does nothing. The [build documentation](https://kurohyou-studios.github.io/k-scaffold/reference/build-api/#k-config) language seems to indicate this should be supported.

**Comment by Kurohyou** [comment]
I'll look into why it fails silently, but the require in .mjs (or .js with a `"type": "module"` in your package.json) is part of the spec for es6 modules.
I'll update the documentation to be more specific about npm modules vs. commonjs packages.

**Comment by Riernar** [comment]
Yeah I figured I was doing something incorrect with JS. For the silence, I suppose `k.All()` gets called with undefined options, or at least doesn't have the `source` directory, and thus does nothing?
If so, a simple warning that no directory to process was passed should be sufficient
